agent/fw_lumen.c

```diff
@@ -104,15 +134,80 @@ NR_PHP_WRAPPER(nr_lumen_name_the_wt) {
 }
 NR_PHP_WRAPPER_END
 
+/*
+ * Exception handling logic. Wraps the function that routes
+ * exceptions to their respective handlers
+ */
+NR_PHP_WRAPPER(nr_lumen_exception) {
+  zval* exception = NULL;
+
+  NR_UNUSED_SPECIALFN;
+  (void)wraprec;
+
+  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_LUMEN);
+#if ZEND_MODULE_API_NO >= ZEND_5_4_X_API_NO
+  const char* class_name = NULL;
+  const char* ignored = NULL;
+#else
+  char* class_name = NULL;
+  char* ignored = NULL;
+#endif /* PHP >= 5.4 */
+
+  char* name = NULL;
+
+  NR_UNUSED_SPECIALFN;
+  (void)wraprec;
+
+  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_LUMEN);
```

Comment:
Why are we requiring the framework a second time?

Reply:
Adding to that.... all three of these lines look to be duplicates.
```
  NR_UNUSED_SPECIALFN;
  (void)wraprec;

  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_LUMEN);
```

Reply:
Great catch; fixed